Plots/ColormapComparisons/CB_Rain.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jun  4 14:12:34 2020
+
```

Comment:
Don't forget to add a description of what this file shows like you did for CB temperature.

Reply:
Just added updates you had mentioned, let me know what you think. Definitely open to feedback/revisions on the description of the plots, it was definitely hard to summarize the goal of creating the plots in only a couple of lines so if any info seems redundant or could be updated let me know!